### ISSUE TITLE
Suggestions on kinetic libraries

### ISSUE THREAD

**Issue by xiaoruiDong** [ReactionMechanismGenerator/RMG-database#349]
1. C3, Fulvene_H, C10H11, vinylCPO_H and napthalene_H have the same descriptions. Maybe Either combine them into one library or tailored descriptions and put them under a folder(like Dooley)
2. 2009_Sharma_C5H5_CH3_highP and c-C5H5_CH3_sharma (Pdep). Maybe put them under a folder and distinguish as high P and Pdep.
3. Glarborg libraries and Methylformate have no descriptions to indicate the source, level of theory used or so on...
4. Mebel_C6H5_C2H2 (Pdep) and /First_to_Second_Aromatic_Ring/2017_Mebel_C6H5_C2H2_highP (highP) are the same except pressure dependency. Maybe distinguish them in the description and put Mebel_C6H5_C2H2 in the /First_to_Second_Aromatic_Ring/
5. Mebel_Napthyl maybe also add to the /First_to_Second_Aromatic_Ring/.
6. Put all the Nitrogen_* together into /Nitrogen?

The organization of the current reaction libraries is a little bit messy, I think we need to clean it up for more user-friendliness. A rough idea is probably we can use the following structure::
----- Fitted Mechanism 
(contain things like GRI, CurrenPentane which are usually 'complete mechanisms and fitted to achieve better performance)
     |
     --- Accurate Submechanisms 
(which contain values we are sure from Ab initio Calculation and Experiments
              |
              --- Submechanisms groups 
             (something like First_to_Second_Aromatic_Rings)
              |
              --- First_to_Second_Aromatic_Rings
              |        |
              |        --- Mebel_C6H5_C2H2
              |               |
              |               --- High P
              |               |
              |               --- Pdep
  .................
Besides, it becomes vaguer to me that how we should use kinetic libraries. We have a tree estimators and a lot of reactions in the kinetic libraries have already added as training reactions so that even if we don't import the kinetic library during the job, we might still use the parameters.

**Comment by mliu49** [comment]
I agree very much with this idea. I think this could be something we do to coincide with the RMG-Py 3.0 release.

The only downside to this is that it breaks compatibility of old input files - you would need to update the names of all the libraries you use.
